maggy/core/experiment_driver/OptimizationDriver.py

```diff
@@ -14,14 +14,19 @@
 #   limitations under the License.
 #
 
+import os
+import time
 import json
 
 from maggy import util
 from maggy.searchspace import Searchspace
-from maggy.optimizer import AbstractOptimizer, RandomSearch, Asha, SingleRun, GridSearch
+from maggy.optimizer import AbstractOptimizer, RandomSearch, Asha, SingleRun
```

Comment:
what happened to gridsearch?

Reply:
I have no clue :D I thought I rebased the branch on the newest version, since it also includes the randomsearch.py optimizer file. I definitely have not deleted it, but maybe something went wrong when I merged the branch into my local copy. Gonna add it back in.